src/plugins/intel_cpu/tests/functional/behavior/ov_executable_network/properties.cpp

```diff
@@ -96,7 +96,7 @@ TEST_F(OVClassConfigTestCPU, smoke_CpuExecNetworkCheckCoreStreamsHasHigherPriori
 
 TEST_F(OVClassConfigTestCPU, smoke_CpuExecNetworkCheckCoreStreamsHasHigherPriorityThanLatencyHint) {
     ov::Core ie;
-    int32_t streams = 4; // latency hint should apply lower number of streams
+    int32_t streams = ov::get_number_of_cpu_cores(); // latency hint should apply lower number of streams
```

Comment:
@Patman11 Is that intended change?

Reply:
I think this may have come from merging master, I did not do those changes.

Reply:
so that did come from a merge before I rebased to apply amended commits with the correct author email. I rebased again to remove that change, and merged from master again. Now it should be good.